plugins/builds/token.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/builds/{id}/token',
+    config: {
+        description: 'Generate a JWT for use throughout a given build',
+        notes: 'Generate a JWT for build using temporal JWT which passed in',
+        tags: ['api', 'builds', 'build_token'],
+        auth: {
+            strategies: ['token'],
+            scope: ['temporal']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const profile = request.auth.credentials;
+            const buildTimeout = request.payload.buildTimeout;
```

Comment:
can we also add a validation here on `buildTimeout` like it's there for executor-base https://github.com/screwdriver-cd/executor-base/pull/42/files#diff-168726dbe96b3ce427e7fedce31bb0bcR138 ?

Reply:
OK! I will add this.

Reply:
I added.